### ISSUE TITLE
Are trackers properly loaded/checked?

### ISSUE THREAD

**Issue by DrLightman** [MailOnline/videojs-vast-vpaid#176]
A VAST specialist testing my platform reported that he only saw this tracker sent:

![trackers](http://h04.imgup.net/trackers07ea3.jpg)

Whereas he expected to see something like this:

![trackers](http://l30.imgup.net/trackers16755.jpg)

If you are interested in looking into this issue I can provide a test url.

**Comment by Fetz** [comment]
Yes, is possible send us the test url. Will take me a bit of time to answer because I'm not working anymore @MailOnline
